tests/beta/test_default_tensor.py

```diff
@@ -685,13 +977,43 @@ def circuit():
             return qml.expval(qml.PauliZ(0))
 
         circuit()
-        state = dev._state
+        state = dev._state()
 
         expected = np.array([[1, 0],
                              [1, 0]]) / np.sqrt(2)
         assert np.allclose(state, expected)
 
 
+    @pytest.mark.parametrize("method", ["auto", "greedy", "branch", "optimal"])
+    def test_correct_state_diff_params(self, tensornet_device_2_wires, method, tol):
```

Comment:
Maybe I'm missing it, but where is `method` used?

Reply:
E.g. in line 1004, 1011?

Reply:
Well spotted, fixed!